src/types/pathways-model.d.ts

```diff
@@ -5,7 +5,7 @@ declare module 'pathways-model' {
     library: string;
     criteria: Criteria[];
     states: {
-      [key: string]: GuidanceState | BranchState;
+      [key: string]: GuidanceState | BranchState | State;
```

Comment:
since `GuidanceState` and `BranchState` both extend `State` this can just be `State`
```suggestion
      [key: string]: State;
```

Reply:
Undoing this commit. Breaks cql-extractor
![Screen Shot 2020-02-17 at 3 35 42 PM](https://user-images.githubusercontent.com/53485517/74684985-33428680-519b-11ea-8d99-1665993c40d2.png)